Gemfile

```diff
@@ -28,22 +28,14 @@ group :system_tests do
   gem 'beaker-puppet_install_helper', :require => false
 end
 
-is_x64 = Gem::Platform.local.cpu == 'x64'
-if is_x64
-  platform(:x64_mingw) do
-    gem "win32-dir", "~> 0.4.9", :require => false
-    gem "win32-process", "~> 0.7.4", :require => false
-    gem "win32-service", "~> 0.8.4", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-  end
-else
-  platform(:mingw) do
-    gem "win32-process", "~> 0.6.5", :require => false
-    gem "win32-service", "~> 0.7.2", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-    if RUBY_VERSION =~ /^1\./ then
-      gem "win32console", :require => false
-    end
+platforms :mswin, :mingw do
```

Comment:
>If you're running on 64-bit ruby, I don't know that it will match either of these.

From the commit message "There is no need to get super specific on x86 versus x64, the Puppet
gem will determine the exact version"

In general the modules don't need to be as specific and can't be if you go across enough versions of Puppet. In all honesty, almost none of this is required anymore once you go above a certain version of Puppet. Once the earliest version we support is Puppet 3.5.x+, we can remove this section entirely.

> rewriting this is actually dangerous based on older Bundler versions.

@Iristyle older Bundler versions won't recognize the `x64_mingw` so I'm not sure I understand. I didn't put back in x64_mingw, so I don't see how what is up for review is dangerous.